src/main/common/utils.ts

```diff
@@ -0,0 +1,13 @@
+import {cloneDeep} from 'lodash';
+
+export default function validateRendering(searchResults,path, req, res, searchInput): void {
+  if (searchResults) {
+    res.render(path, {
+      ...cloneDeep(req.i18n.getDataByLanguage(req.lng)[path]),
+      searchInput : searchInput,
+      searchResults: searchResults,
```

Comment:
you do not need to assign here, you can do like this in typescript:

searchInput,
searchResults,

Reply:
done